wireguard-go-rs/src/lib.rs

```diff
@@ -283,3 +283,7 @@ mod ffi {
         pub fn wgGetSocketV6(handle: i32) -> Fd;
     }
 }
+
+// Make symbols from maybenot-ffi visible
+#[cfg(daita)]
+use maybenot_ffi as _;
```

Comment:
Should this be at the very bottom of the file?

Reply:
Perhaps we should even write more clearly (somewhere) that we are linking `maybenot-ffi` to the app only for the symbols to be visible to  `wireguard-go`, not for the app itself, instead of building it into the `wiregurad-go` archive using the makefile.

Reply:
Also, we should document that the `maybenot-ffi` version in the toml file has to match the version checked in to `wireguard-go`